### ISSUE TITLE
Gaussholder with ajax loaded items

### ISSUE THREAD

**Issue by newcoderforwhat** [humanmade/Gaussholder#67]
If I load some posts via ajax and insert these items on the page, Gaussholder doesn't work. How can I call some kind of callback to reinitialize Gaussholder for dynamically added objects?

**Comment by newcoderforwhat** [comment]
This is possible with a call to DOMContentLoaded. I don't think it's the best option, but it works well.
`            window.document.dispatchEvent(new Event("DOMContentLoaded", {
              bubbles: true,
              cancelable: true
            }));`

**Comment by rmccue** [comment]
I believe currently the Gaussholder functions aren't exposed outside of the event handler, but would be happy to accept a PR to add that; I imagine it might need to avoid re-initialisating existing images as well.

**Comment by roborourke** [comment]
Fixed in #71 - you can now use the `GaussHolder();` function in the window context, for example:

```js
wp.apiFetch( { path: '/wp/v2/posts' } ).then( ( posts ) => {
    console.log( posts );
    window.GaussHolder && GaussHolder();
} );
```
